gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/transformer/RequestPreProcessorTest.java

```diff
@@ -56,16 +56,18 @@ public void teardown() {
   @Test
   public void testServiceEntitiesRequestDuplicateColumnSelectionIsRemoved() {
     initializeDomainObjectConfigs("configs/request-preprocessor-test/service-id-config.conf");
+    long endTime = System.currentTimeMillis();
+    long startTime = endTime - 1000L;
     EntitiesRequestContext entitiesRequestContext =
-        new EntitiesRequestContext(TEST_TENANT_ID, 0L, 1L, "SERVICE", Map.of());
+        new EntitiesRequestContext(TEST_TENANT_ID, startTime, endTime, "SERVICE", "SERVICE.startTime", Map.of());
     mockAttributeMetadata(entitiesRequestContext, AttributeScope.SERVICE.name(), "id");
     mockAttributeMetadata(entitiesRequestContext, AttributeScope.SERVICE.name(), "startTime");
 
     EntitiesRequest entitiesRequest =
         EntitiesRequest.newBuilder()
             .setEntityType("SERVICE")
-            .setStartTimeMillis(0L)
-            .setEndTimeMillis(1L)
+            .setStartTimeMillis(startTime)
```

Comment:
What's wrong with 0 and 1

Reply:
Nothing wrong. I mainly wanted to pull out the start and end times into variables so that I can reuse and I just assigned real timestamp values to them while I was there.